src/components/form/Editable.js

```diff
@@ -1,58 +1,75 @@
 import React, { useState, useEffect, useRef } from 'react'
 import styled from 'styled-components'
+import PropTypes from 'prop-types'
 
-export default function Editable({ value, onSubmit, type = 'text' }) {
+Editable.propTypes = {
+  label: PropTypes.string.isRequired,
+  value: PropTypes.any.isRequired,
+  onSubmit: PropTypes.func.isRequired,
+  type: PropTypes.string
+}
+Editable.defaultProps = {
+  type: 'text'
+}
+export default function Editable({ value, onSubmit, label, type = 'text' }) {
```

Comment:
Reicht es nicht, den default-wert entweder in den default-props oder in den funktions-parametern zu setzen?